polyhost/main_app.py

```
@@ -18,32 +18,163 @@ def main():
+                    extra_args=_spawned_daemon_flags(args),
+                    log=logging.getLogger("PolyHost"))
+                if spawned is not None and dl.wait_until_live():
+                    print("Core daemon is up; attaching as a client.")
+                    client_mode, endpoint, daemon_handled_instance = True, None, True
+                else:
+                    print("Core daemon did not come up; running in-process instead.")
+                    if spawned is not None:
+                        try:
+                            spawned.terminate()
+                        except Exception:
+                            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)